zone/attack.cpp

```diff
@@ -4231,8 +4272,35 @@ void Mob::HealDamage(uint64 amount, Mob *caster, uint16 spell_id)
 						caster->FilteredMessageString(caster, Chat::NonMelee, FilterHealOverTime,
 							YOU_HEAL, GetCleanName(), itoa(acthealed));
 				}
+
+				if (caster->IsBot() && this != caster->CastToBot()->GetBotOwner() && RuleB(Bots, DisplayHealDamage)) {
+
+
+					// %1 healed %2 for %3 hit points from %4's %5
+					const char s2[]{ " healed " };
+					const char s4[]{ " for " };
+					const char s6[]{ " hit points from " };
+					const char s8[]{ "'s " };
+
+					caster->CastToBot()->GetBotOwner()->FilteredMessageString(
+						caster->CastToBot()->GetBotOwner(), //send to the Bot Owner's client
+						Chat::NonMelee,
+						FilterHealOverTime,
+						GENERIC_9_STRINGS,					//using generic for testing purposes %1 %2 %3 %4 %5 %6 %7 %8 %9
+						caster->GetCleanName(),				// %1 caster (bot's) name
+						s2,									// %2
+						this->GetCleanName(),				// %3 caster (bot's) target
+						s4,									// %4
+						itoa(acthealed),					// %5 amount healed
+						s6,									// %6
+						caster->GetCleanName(),				// %7 caster (bot's) name
+						s8,									// %8
+						spells[spell_id].name				// %9 spell name
+					);
```

Comment:
We can use fmt::format() here if we need to or see if there's an `eqstr_us.txt` entry.

Reply:
I could not find an entry that gave me sufficient flexibility for %1 healed %2 for %3 hit points from %4's %5.  This is really to give fidelity for any parsers out there.